docs/Gemfile

```diff
@@ -0,0 +1,14 @@
+# Gemfile
```

Comment:
Nor this one.

Reply:
ditto

Reply:
Does not requiere too much maintaining as we have already see at kubernetes-nmstate, usually just calling `make update` sometimes is enough.